runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -201,7 +201,7 @@ TR::S390J9CallSnippet::generateInvokeExactJ2IThunk(TR::Node * callNode, int32_t
       *(uint32_t *) cursor = 0xe3000006 + finalCallLength + (rEP << 12) + (rEP << 20);  // LG      rEP,8(,rEP)
       cursor += 4;
       *(uint16_t *) cursor = 0x0004;
-      cursor += 2;sizeof(int16_t);
+      cursor += 2;
```

Comment:
I imagine the intent was a comment:
```c
    cursor += 2; // sizeof(int16_t);
```
Perhaps better would be to use `sizeof` instead of `2`:
```c
    cursor += sizeof(int16_t);
```

Reply:
done